src/registrar/admin.py

```diff
@@ -3104,6 +3104,83 @@ def get_non_admin_users(self, obj):
 
         return non_admin_users
 
+    def display_admins(self, obj):
```

Comment:
Who removed/why was this removed from main? This code came back when I merged main. cc @abroddrick @lizpearl

Reply:
Actually, I think it's a @zandercymatics refactor

Reply:
Got it